src/config.ts

```diff
@@ -18,19 +36,23 @@ export const LOCALSTACK_ENDPOINT: string | undefined = loadOptionalEnvVar(
 
 // Zitadel
 
+export const ZITADEL_DOMAIN: string = loadRequiredEnvVar("ZITADEL_DOMAIN");
+
 export const ZITADEL_APPLICATION_KEY: string = loadRequiredEnvVar(
   "ZITADEL_APPLICATION_KEY",
 );
 
-export const ZITADEL_DOMAIN: string = loadRequiredEnvVar("ZITADEL_DOMAIN");
-
 // Internal function to load environment variables
 
 function loadOptionalEnvVar(envVarName: string): string | undefined {
   return process.env[envVarName];
 }
 
 function loadRequiredEnvVar(envVarName: string): string {
+  if (process.env.NODE_ENV === "test") {
+    return "TEST_IN_PROGRESS";
```

Comment:
Add to re implement this condition as the way we mock the configuration variables during Vitest setup has been implemented in a way that will load the config file (`importOriginal`) as is before altering its content for testing purpose. Since the code in the config file will be executed it will try to load variables that don't exist in the Github Action environment.